appveyor.yml

```diff
@@ -26,6 +26,15 @@
     - cmd: >-
         subst s: c:\projects
     - ps: set-location -path s:\node_modules\node-sass
+    - ps: Uninstall-Product node 4 $env:platform
+    - ps: Write-Host "Installing node-chakracore..." -ForegroundColor Cyan
+    - ps: Write-Host "Downloading..."
+    - ps: $msiPath = "$env:USERPROFILE\node-chakracore-$env:platform.msi"
+    - ps: $file = "https://github.com/nodejs/node-chakracore/releases/download/node-chakracore-7.0.0-pre9/node-chakracore-$($env:platform).msi"
```

Comment:
>I am confused about what was the intent of making .appveyor.yml changes as part of this PR

It was the best way IMHO to verify this PR is working as expected. As I originally stated we weren't willing to have hacks for node-chakracore when it doing the right way was just as easy.

> Was it a prep work so when node-chakracore binaries become available

Correct. Also having CI working with chakracore gives us confidence in support it.